elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -1,79 +1,140 @@
-  --pfe-cta__inner--BorderColor:      transparent;
-  --pfe-cta--Color:   #{pfe-color(ui-link)};
-  --pfe-cta--Padding: .6rem 0;
 
-  // hover variables
-  --pfe-cta--BackgroundColor--hover:  transparent;
-  --pfe-cta--BorderColor--hover: transparent;
-  --pfe-cta--Color--hover:  #{pfe-color(ui-link--hover)};
+  --pfe-cta--BorderColor:           transparent;
+  --pfe-cta--BorderColor--hover:    transparent;
+  --pfe-cta--BorderColor--focus:    transparent;
 
-  // focus variables
-  --pfe-cta--BorderColor--focus: transparent;
+  --pfe-cta__inner--BorderColor:        transparent;
   --pfe-cta__inner--BorderColor--focus: transparent;
-  --pfe-cta--Color--focus:  #{pfe-color(ui-link--focus)};
+
+  --pfe-cta--Color:          #{pfe-radio(ui-link)};
+  --pfe-cta--Color--hover:   #{pfe-radio(ui-link, hover)};
+  --pfe-cta--Color--focus:   #{pfe-radio(ui-link, focus)};
+
+  --pfe-cta--TextDecoration:          none;
+  --pfe-cta--TextDecoration--hover:   none;
+  --pfe-cta--TextDecoration--focus:   none;
+  --pfe-cta--FontWeight:              #{pfe-var(font-weight--bold)};
 
   // properties
-  display: inline-block;
-  font-family: #{pfe-var(font-family--heading)};
-  font-size:   #{pfe-var(font-size)};
-  font-weight: #{pfe-var(font-weight--bold)};
-  line-height: #{pfe-var(line-height)};
+  display:         inline-block;
+  position:        relative;
+  vertical-align:  middle;
+  transition:      padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
+
   background-color: #{pfe-local(BackgroundColor)};
   border-color:     #{pfe-local(BorderColor)};
-  position: relative;
-  transition: .3s ease padding; // padding #{pfe-var(animation-timing)};
-  line-height: inherit;
-  vertical-align:  middle;
+
   background-color: #{pfe-local(BackgroundColor)};
   border-radius:    #{pfe-local(BorderRadius)};
   border:           #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor)};
+  padding:          #{pfe-local($cssvar: Padding)};
+
+  ::slotted(button),
+  ::slotted(input) {
+    @include pfe-reset-button;
+  }
+
+  ::slotted(*) {
+    display: inline;
+
+    color:           #{pfe-local(Color, $pfe-cta--Color--fallback)} !important;
+    font-family: #{pfe-local($cssvar: FontFamily, $fallback: #{pfe-var(font-family--heading)})};
+    font-size:   #{pfe-local($cssvar: FontSize, $fallback: #{pfe-var(font-size)})};
+    font-weight: #{pfe-local(FontWeight)};
+    line-height: #{pfe-local($cssvar: LineHeight, $fallback: #{pfe-var(line-height)})};
+    text-decoration: #{pfe-local(TextDecoration)} !important;
+    
+    &::before {  // inner border on priority CTAs
+      content: "";
+
+      display:    block;
+      box-sizing: border-box;
+      position:   absolute;
+      top:    2px;
+      left:   2px;
+      bottom: 2px;
+      right:  2px;
+
+      border: #{pfe-var(surface--border-width)} #{pfe-var(surface--border-style)} #{pfe-local($cssvar: BorderColor, $region: inner)};
```

Comment:
same here, i think this needs to use the "inner border color"

Reply:
Yup, that's the 3rd item in the border stack here: `#{pfe-local($cssvar: BorderColor, $region: inner)}` == `var(--pfe-cta__inner--BorderColor)`